tests/bluetooth/tester/src/bttester.h

```diff
@@ -918,6 +918,350 @@ struct mesh_lpn_unsubscribe_cmd {
 
 #define MESH_RPL_CLEAR			0x12
 #define MESH_PROXY_IDENTITY		0x13
+#define MESH_COMP_DATA_GET		0x14
+struct mesh_comp_data_get {
```

Comment:
every cmd sould end with _cmd suffix. So in this example `mesh_comp_data_get_cmd`. Look at other structures.